tools/__tasks__/compile/conf/clean.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+const rimraf = require('rimraf');
+
+const {root} = require('../../config').paths;
+
+module.exports = {
+    description: 'Clear template rendering artefacts',
+    task: () => rimraf.sync(path.resolve(root, 'common', 'conf', 'assets'))
```

Comment:
> My thought just now was that every task should have a clean and a build, ideally.

not sure, i think the less prescriptive it is, the better. certainly would ecnourage tasks discretely preparing their own ground. i guess the grunt set up was the result of grunt tasks being per-plugin, rather than per-task